templates/addons/windows/calico/kube-proxy-windows.yaml

```diff
@@ -4,7 +4,7 @@ metadata:
   labels:
     k8s-app: kube-proxy
   name: kube-proxy-windows
-  namespace: calico-system
+  namespace: kube-system
```

Comment:
this changed in #2495, I thought I noticed in my testing that kube-proxy needed to be in the same namespace as the calico daemonset for calico to work with windows, were you able to get a working cluster / passing test with this?

Reply:
I got a working cluster with this change, Haven't run all the test yet.